src/ui/public/management/section.js

```diff
@@ -20,10 +23,18 @@ export default class ManagementSection {
       index: ['id'],
       order: ['order']
     });
+    this.visible = true;
+    this.disabled = false;
+    this.tooltip = '';
+    this.url = '';
 
     assign(this, options);
   }
 
+  get visibleItems() {
+    return this.items.inOrder.filter(item => item.visible);
```

Comment:
Jim checked, and `inOrder` and `byVisible` both return simple arrays and therefore can not be chained. So this is fine.